Actons  bug

ERROR: failed to solve: process "/bin/sh -c composer update --no-dev" did not complete successfully: exit code: 2
Unable to find image 'cwcc-spc-x86_64:latest' locally
docker: Error response from daemon: pull access denied for cwcc-spc-x86_64, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
Error: Process completed with exit code 125.

--- a/bin/spc-alpine-docker
+++ b/bin/spc-alpine-docker
@@ -64,7 +64,6 @@ RUN apk update; \
         bison \
         build-base \
         cmake \
-        composer \
         curl \
         file \
         flex \
@@ -77,16 +76,20 @@ RUN apk update; \
         linux-headers \
         m4 \
         make \
-        php81 \
-        php81-common \
-        php81-pcntl \
-        php81-phar \
-        php81-posix \
-        php81-tokenizer \
-        php81-xml \
+        php82 \
+        php82-common \
+        php82-pcntl \
+        php82-phar \
+        php82-posix \
+        php82-tokenizer \
+        php82-dom \
+        php82-xml \
+        php82-xmlwriter \
+        composer \
         pkgconfig \
         wget \
         xz
+RUN ln -s /usr/bin/php82 /usr/bin/php
 WORKDIR /app
 ADD ./src /app/src
 ADD ./composer.json /app/composer.json

